README.md

```diff
@@ -1,15 +1,76 @@
-# Fast Store
+# FastStore 
+## An open ecommerce framework
 
-https://docs.google.com/presentation/d/10SCY7ZO1WWGeUQs6VzJWAlp4r1404dVtIxx_YgYldZI/edit#slide=id.g8ef796eccd_0_336
+FastStore is an open ecommerce framework built using modern technologies that aims to deliver the building blocks necessary for creating your custom ecommerce experience. With these building blocks you can archieve:
 
-https://miro.com/app/board/o9J_kobcu-Y=/
-This is the home for the proof of concept project to create VTEX sites based on the Gatsby framework.
+- Multiregion stores with native internationalization support
+- Great Performance (90+ on Google's page speed insights)
+- SEO ready pages with Google's Rich Results support
+- CMS managed content
+- Jamstack based (deploy with Netlify, Vercel and any other JAMStack platform)
+- Marketplace ready with support to millions of skus
+- Intelligence via Google Analytics and GTM
+- Evergreen stores with our no-fork based approach
 
-# How to develop
+## Why building FastStore?
+The web is a wild and noisy environment. Many projects claims to solve many different aspects of web development but none of them solves the ecommerce problem satisfactory. Ecommerce is a very complex case where performance is a must, but personalization is the key. 
+
+After years of developing ecommerces, FastStore creators packaged all of this knowledge into easy, ready to use building blocks for crafting production ready, multi-region marketplaces.
+
+FastStore not only contains code, but knowledge and guidelines to change the way you think and build ecommerces
+
+## Project philosophy
+This project has a very well defined list of priorities. When solving an issue, a lower ranking priority must not degrade or override an upper ranking priority.
+The priorities are:
+
+1. Performance
+2. Stability - e2e tests are complete and make sure the critical paths are covered
+3. No fork - Features should be delivered by dependencies and not by a store fork
+4. Feature completion
+
+## Why Jamstack and Gatsby?
+The main points of Jamstack are performance and stability. This is very well aligned to our core philosophy. 
+
+Gatsby has a very mature plugin system where the `No Fork` philosophy can be implemented via Gatsby Themes and Shadowing. Also, Gatsby is very aligned with performance by implementing the PRPL architecture.
+
+## Continuous Performance and Stability
+Maintaining the FastStore performance and stability is a key element. For this, using Lighthouse CI and Cypress in your CI/CD pipeline is recommended. 
+
+We provide a base setup for both platforms so you can easily add them to your CI/CD platform.
+
+## Docs
+We are still missing many docs and guides, however some of them are available at the `docs` folder. Available docs are:
```

Comment:
"Store Framework Jamstack docs are under development. For now, the available docs are:"